keps/sig-node/1769-memory-manager/README.md

```diff
@@ -586,20 +658,34 @@ the health of the service?**
     To understand the reason you will need to check via pod resources API 
     the amount of allocatable memory and memory reserved by containers.
 
-* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
   This does not seem relevant to this feature.
```

Comment:
We might have to fix this. I think we will need point to a latency related metric in order to ensure that allocation of memory by memory manager is  not taking longer than expected.

Reply:
On one hand I agree with you that this could help, OTOH I don't recall having this for device manager and cpu manager. Perhaps we want to add this metric to the memory manager and fix the other managers in a followup (and non-GA blocking) PR?